dali/operators/reader/file_reader_op.cc

```diff
@@ -98,6 +107,13 @@ This argument is mutually exclusive with ``file_list``.)", nullptr)
 ``files`` argument.
 
 If not used, sequential 0-based indices are used as labels)", nullptr)
+  .AddOptionalArg<string>("file_filters", R"(A list of glob strings to filter the
+list of files in the sub-directories of the ``file_root``.
+
+This argument is ignored when file paths are taken from ``file_list`` or ``files``.)",
+      kKnownExtensionsGlob)
+  .AddOptionalArg<bool>("case_sensitive_filter", R"(If set to True, the filter will be applied
+to original file paths, otherwise to lower-cased file paths.)", false)
```

Comment:
This is fine, but I think it'd be even better if we just make the filter case insensitive as well. That is, case insensitive `filter="*.JPEG"` would match `"dog.jpeg"`